File: netsend/src/NetSendPacket.java
public class NetSendPacket {
  public double tick;
  public double turtleID;
  public String name;
  public double data;

  public NetSendPacket (double tick, double turtleID, String name, double data) {
    this.tick = tick;
    this.turtleID = turtleID;
    this.name = name;
    this.data = data;
  }

  public String toString() {
    String str = "";
    str += tick + "\t";
    str += turtleID + "\t";
    str += name + "\t";
    str += data;

    return str;
  }
}

File: other/MarneMap/MarneMap/NetSendPacket.m
//
//  NetSendPacket.m
//  MarneMap
//
//  Created by Dylan Garrett on 4/8/12.
//  Copyright (c) 2012 __MyCompanyName__. All rights reserved.
//

#import "NetSendPacket.h"

@implementation NetSendPacket
@synthesize tick, turtleID, name, data;

- (id)initWithTick:(NSNumber*)newTick turtleID:(NSNumber*)newID name:(NSString*)newName data:(NSNumber*)newData
{
    if (self = [super init]) {
        self.tick = newTick;
        self.turtleID = newID;
        self.name = newName;
        self.data = newData;
    }
    
    return self;
}

- (NSString*)description
{
    return [NSString stringWithFormat:@"tick: %@, name: %@, data: %@", self.tick, self.name, self.data];
}

@end
